src/components/Sidebar/contentTreeNode.js

```diff
@@ -122,25 +122,32 @@ const ContentTreeNode = ({ className, toggle, collapsed, url, title, location, c
       location.pathname === url + '/' ||
       location.pathname === config.metadata.pathPrefix + url);
   const collapse = () => {
+    console.log("test click", url);
+    if(typeof toggle !== "function") return;
```

Comment:
Could you please add a comment above this line why we need to check if is a function and not null?
This is because we haven't check it before and it is very interesting.

Reply:
Yes